queries-api/src/main/java/io/corbel/lib/queries/parser/JacksonQueryParser.java

```diff
@@ -115,6 +119,11 @@ private QueryNodeImpl getQueryNode(QueryOperator operator, JsonNode nodeField) t
     }
 
     @SuppressWarnings("rawtypes")
+    private QueryLiteral<?> generateObjectQueryLiteral(JsonNode nodeField) throws MalformedJsonQueryException {
+        return new PositionQueryLiteral(jsonParser, nodeField);
```

Comment:
Yes, you're right. I thought about changing when I had already created the pull request :)